diff -U 3 -H -d -r -N -- lxqt-common-0.10.0-old/config/lxqt.conf lxqt-common-0.10.0/config/lxqt.conf
--- lxqt-common-0.10.0-old/config/lxqt.conf	2015-10-09 00:38:12.000000000 +0800
+++ lxqt-common-0.10.0/config/lxqt.conf	2015-12-11 19:45:19.290549659 +0800
@@ -9,3 +9,6 @@
 doubleClickInterval=400
 style=Fusion
 wheelScrollLines=3
+
+[Mouse]
+cursor_theme=Oxygen_White
diff -U 3 -H -d -r -N -- lxqt-common-0.10.0-old/config/pcmanfm-qt/lxqt/settings.conf.in lxqt-common-0.10.0/config/pcmanfm-qt/lxqt/settings.conf.in
--- lxqt-common-0.10.0-old/config/pcmanfm-qt/lxqt/settings.conf.in	2015-10-09 00:38:12.000000000 +0800
+++ lxqt-common-0.10.0/config/pcmanfm-qt/lxqt/settings.conf.in	2015-12-11 19:46:21.963654542 +0800
@@ -4,8 +4,8 @@
 TerminalCommand=
 Archiver=file-roller
 SIUnit=false
-TerminalDirCommand=xterm
-TerminalExecCommand=xterm -e %s
+TerminalDirCommand=qterminal-qt5
+TerminalExecCommand=qterminal-qt5 -e %s
 
 [Behavior]
 BookmarkOpenMethod=0
diff -U 3 -H -d -r -N -- lxqt-common-0.10.0-old/startlxqt.in lxqt-common-0.10.0/startlxqt.in
--- lxqt-common-0.10.0-old/startlxqt.in	2015-10-09 00:38:12.000000000 +0800
+++ lxqt-common-0.10.0/startlxqt.in	2015-12-11 19:48:25.111860617 +0800
@@ -62,7 +62,7 @@
 # use lxqt-applications.menu for main app menu
 export XDG_MENU_PREFIX="lxqt-"
 
-export XDG_CURRENT_DESKTOP="LXQt"
+export XDG_CURRENT_DESKTOP="X-LXQt"
 
 # Start the LXQt session
 exec lxqt-session
