diff -U 3 -dHrN -- lxqt-common-0.11.0.orig/config/lxqt.conf lxqt-common-0.11.0/config/lxqt.conf
--- lxqt-common-0.11.0.orig/config/lxqt.conf	2016-09-24 06:44:22.000000000 +0800
+++ lxqt-common-0.11.0/config/lxqt.conf	2016-10-16 11:59:25.222694804 +0800
@@ -1,11 +1,12 @@
 [General]
-theme=frost
+theme=fedberry
 icon_theme=oxygen
 single_click_activate=false
 tool_button_style=ToolButtonTextBesideIcon
 
 [Qt]
-font="Sans,11,-1,5,50,0,0,0,0,0"
+fixedFont="Monospace,10,-1,5,50,0,0,0,0,0"
+font="Sans,10,-1,5,50,0,0,0,0,0"
 doubleClickInterval=400
 style=Fusion
 wheelScrollLines=3
diff -U 3 -dHrN -- lxqt-common-0.11.0.orig/config/pcmanfm-qt/lxqt/settings.conf.in lxqt-common-0.11.0/config/pcmanfm-qt/lxqt/settings.conf.in
--- lxqt-common-0.11.0.orig/config/pcmanfm-qt/lxqt/settings.conf.in	2016-09-24 06:44:22.000000000 +0800
+++ lxqt-common-0.11.0/config/pcmanfm-qt/lxqt/settings.conf.in	2016-10-18 09:17:27.131790364 +0800
@@ -1,11 +1,11 @@
 [System]
 IconThemeName=elementary
 SuCommand=lxqt-sudo %s
-TerminalCommand=
-Archiver=file-roller
+TerminalCommand=qterminal
+Archiver=xarchiver
 SIUnit=false
-TerminalDirCommand=xterm
-TerminalExecCommand=xterm -e %s
+TerminalDirCommand=qterminal
+TerminalExecCommand=qterminal -e %s
 
 [Behavior]
 BookmarkOpenMethod=0
@@ -15,9 +15,9 @@
 ConfirmDelete=true
 
 [Desktop]
-WallpaperMode=stretch
-Wallpaper=@LXQT_SHARE_DIR@/themes/frost/lxqt-origami-light.png
-BgColor=#000000
+WallpaperMode=zoom
+Wallpaper=/usr/share/backgrounds/default.png
+BgColor=#333333
 FgColor=#ffffff
 ShadowColor=#000000
 ShowHidden=false
diff -U 3 -dHrN -- lxqt-common-0.11.0.orig/config/session.conf lxqt-common-0.11.0/config/session.conf
--- lxqt-common-0.11.0.orig/config/session.conf	2016-09-24 06:44:22.000000000 +0800
+++ lxqt-common-0.11.0/config/session.conf	2016-10-18 09:17:03.801733107 +0800
@@ -5,10 +5,12 @@
 [Environment]
 GTK_CSD=0
 GTK_OVERLAY_SCROLLING=0
+BROWSER=qupzilla
+TERM=qterminal
 
 [Mouse]
 cursor_size=18
-cursor_theme=whiteglass
+cursor_theme=Breeze_Snow
 acc_factor=20
 acc_threshold=10
 left_handed=false
diff -U 3 -dHrN -- lxqt-common-0.11.0.orig/startlxqt.in lxqt-common-0.11.0/startlxqt.in
--- lxqt-common-0.11.0.orig/startlxqt.in	2016-09-24 06:44:22.000000000 +0800
+++ lxqt-common-0.11.0/startlxqt.in	2016-10-16 12:04:01.044560415 +0800
@@ -91,7 +91,7 @@
 # use lxqt-applications.menu for main app menu
 export XDG_MENU_PREFIX="lxqt-"
 
-export XDG_CURRENT_DESKTOP="LXQt"
+export XDG_CURRENT_DESKTOP="X-LXQt"
 
 # Start the LXQt session
 exec lxqt-session
